src/models/commands.ts

```diff
@@ -419,6 +424,60 @@ export const fcCommands: FCCommand[] = [
 			return dxService.openOrgPage('/apex/' + vfFileName);
 		}
 	},
+	{
+		commandName: 'ForceCode.openFlowBuilder',
+		hidden: true,
+		command: function (context, selectedResource?) {
+			commandService.runCommand('ForceCode.launchFlowBuilder', context, selectedResource);
+		}
+	},
+	{
+		commandName: 'ForceCode.launchFlowBuilder',
+		hidden: true,
+		name: 'Opening Flow Builder',
+		command: async function (context, selectedResource?, task?: Task) {
+			return new Promise((resolve, reject) => {
+				fs.readFile(context.fsPath, (err, data) => {
+					if(err) reject(err);
+					parseString(data, (err, result) => {
+						if(err) reject(err);
+						else {
+							const label = result.Flow.label[0];
+							const query = `SELECT Id FROM Flow WHERE MasterLabel = '${label}' ORDER BY VersionNumber DESC`;
+							vscode.window.forceCode.conn.tooling.query(query).then(res => {
+								let options = {
+									wait: true,
+									app: null
+								};
+
+								let randomDir = fs.mkdtempSync(path.join(tmpdir(), 'forcecode-'));
+
+								switch(process.platform) {
+									case 'darwin':
+										options.app = ['google chrome', `--user-data-dir=${randomDir}`];
+										break;
+									case 'win32':
+										options.app = ['chrome', `--user-data-dir=${randomDir}`];
+										break;
+									case 'linux':
+										options.app = ['google-chrome', `--user-data-dir=${randomDir}`];
+										break;
+								}
+
+								dxService.getOrgPageUrl(`/builder_platform_interaction/flowBuilder.app?isFromAloha=false&flowId=${res.records[0].Id}`).then(response => {
+									CommandViewService.getInstance().updateLabel(task, `Editing ${label}`);
+									opn(response.url, options).then(child => {
+										resolve(commands.retrieve(context, task));
```

Comment:
Will this actually wait until the user is done using Flow Builder before it resolves? It's a really neat idea, but I don't like the fact that it only works with Chrome. Maybe a section in the readme for the new option would be a good idea, and to let users know it will only work with chrome.